.evergreen/csfle/await_servers.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# wait for the KMS servers to start.
+set -eu
+
+SCRIPT_DIR=$(dirname ${BASH_SOURCE[0]})
+. $SCRIPT_DIR/../handle-paths.sh
+pushd $SCRIPT_DIR
+
+# Wait until the pids file has been created.
+echo "Waiting for servers to start..."
+await_pidfile() {
+    for i in $(seq 10); do
```

Comment:
```suggestion
    for i in $(seq 300); do
```

To address timeout observed in https://spruce.mongodb.com/task/mongo_c_driver_cse_matrix_openssl_test_start_servers_patch_1eacb5d46f39c5404586e5d3d32e4bf9e6670bd2_65b2b7b70ae6068c7a73b88a_24_01_25_19_34_16/logs?execution=0:
```
[2024/01/25 14:35:36.548] Could not detect PID file
```